cost/aws/unused_rds/unused_rds.pt

```diff
@@ -288,19 +310,16 @@ end
 ###############################################################################
 
 escalation "email" do
+  automatic true
+  label "Send Mail"
+  description "Sends incidents email"
   email $param_email
 end
 
 escalation "decommision_rds" do
-  request_approval  do
-    label "Escalation approval"
-    description "Approve Unused RDS decommission action"
-    parameter "approval_reason" do
-      type "string"
-      label "Reason for approval"
-      description "Explain why you are approving the action"
-    end
-  end
+  automatic false
+  label "Decommission Unused RDS"
```

Comment:
We should standardize on action names.  please change this to Delete RDS Instance, and update the README with the delete changes.